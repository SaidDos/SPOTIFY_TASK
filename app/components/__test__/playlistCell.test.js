import 'react-native';
import React from 'react';
import PlaylistCell from '../cells/playlistCell';

import renderer from 'react-test-renderer';

it('should renders correctly and generating a snapShot', () => {
  let playlist = {name: 'said', image: 'someImage'};
  const tree = renderer.create(<PlaylistCell playlist={playlist} />).toJSON();
  expect(tree).toMatchSnapshot();
});
