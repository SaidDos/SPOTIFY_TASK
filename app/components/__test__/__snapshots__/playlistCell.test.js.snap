// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly and generating a snapShot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "lightgrey",
      "flexDirection": "row",
      "height": 70,
      "marginBottom": 20,
      "padding": 10,
      "width": 319.3,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "someImage",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "height": 55,
        "justifyContent": "flex-start",
        "width": 55,
      }
    }
  />
  <View
    style={
      Object {
        "marginHorizontal": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 16,
          "fontWeight": "bold",
        }
      }
    >
      said
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
        }
      }
    >
       by 
    </Text>
  </View>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 30,
        },
        Object {
          "alignSelf": "center",
          "position": "absolute",
          "right": 10,
        },
        Object {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    
  </Text>
</View>
`;
